e2e/cypress/integration/integrations/incoming_webhook/slack_formatting_spec.js

```diff
@@ -0,0 +1,325 @@
+
+        cy.get(`#sidebarItem_${testChannel.name}`).find('#unreadMentions').should('have.text', '1');
+        cy.get(`#sidebarItem_${testChannel.name}`).click();
+
+        cy.getLastPost().within((el) => {
+            cy.get('.attachment__thumb-pretext').should('contain', id);
+            cy.get('.attachment__thumb-pretext span[data-emoticon="eyes"]').should('exist');
+
+            cy.get('.attachment__body .post-message__text-container a.mention-link').should('have.text', '@sysadmin');
+            cy.get('a.attachment__title-link[href="https://www.google.com"]').should('have.text', 'A slack attachment');
+            cy.get('.attachment-field a.markdown__link[href="http://example.com"]').should('have.text', 'Link');
+            cy.get('.attachment-field span[data-emoticon="taco"]').should('exist');
+
+            cy.get('.attachment__thumb-container .file-preview__button img').should('exist');
+        });
+    });
+
+    describe('MM-T631 Short field in payload can accept strings text in quotes for true and false', () => {
+        const id = 'MM-T631';
+
+        const makePayloadFromShortValue = (short, currentID) => ({
+            channel: testChannel.name,
+            "attachments": [{"type": "slack_attachment", "color": "#7CD197", "fields": [{"short": false, "title": "Area", "value": "Testing with a very long piece of text that will take up the whole width of the table (stopping short of the space where the thumbnail image is displayed). This is one more sentence to really make it a long field, and let's add a taco emoji :taco:."}, {"short": true, "title": "Iteration", "value": "Testing"}, {"short": true, "title": "State", "value": "New"}, {"short": false, "title": "Reason", "value": "New defect reported"}, {"short": false, "title": "Random field", "value": "This is a field which is not marked as short so it should be rendered on a separate row"}, {"short": true, "title": "Short 1", "value": "Short field"},
+            {"short": short, "title": "Short 2", "value": "Another one"}, {"short": true, "title": "Field with link", "value": "<http://example.com|Link>"}], "mrkdwn_in": ["pretext"], "pretext": "Some text here to look at (verify eyes emoji) :eyes:",
+            "text": `${currentID} ${short} This is the text of the attachment. <@${sysadminUser.id}>, there should be a small Jenkins thumbnail off to the right.`, "thumb_url": "https://slack.global.ssl.fastly.net/7bf4/img/services/jenkins-ci_128.png", "title": "A slack attachment", "title_link": "https://www.google.com"
+        }]});
+
+        const testCases = [
+            {short: true, shouldShowShort: true},
+            {short: false, shouldShowShort: false},
+            {short: 'true', shouldShowShort: true},
+            {short: 'false', shouldShowShort: false},
+        ];
+
+        testCases.forEach((testCase, i) => {
+            it('should show table elements based on short value ${i}', () => {
```

Comment:
Looks like this should be
```suggestion
            it(`should show table elements based on short value ${i}`, () => {
```